### ISSUE TITLE
请问可以提供下/mnt/data/seq2vec文件夹里的文件吗

### ISSUE THREAD

**Issue by yuanpf123** [ZhangWeihang99/HVSA#3]
如下配置
    seq2vec:
        arch: skipthoughts
        dir_st: /mnt/data/seq2vec
        type: BayesianUniSkip
        dropout: 0.25
        fixed_emb: False

**Comment by ZhangWeihang99** [comment]
有关 seq2vec 的文件已在 README.md 中声明。
“Download the dataset files and pre-trained models. Files about seq2vec are from [here](https://github.com/xiaoyuan1996/AMFMN/tree/master/AMFMN).”

Files about Seq2Vec are declared in the README.md. 
“Download the dataset files and pre-trained models. Files about seq2vec are from [here](https://github.com/xiaoyuan1996/AMFMN/tree/master/AMFMN).”
